satellite/gc/service.go

```diff
@@ -164,3 +168,9 @@ func (service *Service) lastPieceCountsNumNodes() int {
 
 	return len(m.(map[storj.NodeID]int))
 }
+
+func (service *Service) isActiveFrom(from time.Time) bool {
```

Comment:
should change to isEnabledFrom

Reply:
I named this method intentionally like this because I believed that `isEnabled` was going to look like only checking the `Enabled` config field.

However, the method evolved to be `isActiveFrom` hence `isEnabledFrom` may not cause the confusion anymore.

Would you like that we rename it despite of the reason that I've written down (I don't mind to rename it if that's looks clear)?

Reply:
okay I think the justification makes sense, isActiveFrom is fine!